eng/helix/content/RunTests/TestRunner.cs

```diff
@@ -322,21 +323,19 @@ public void UploadResults()
             {
                 Console.WriteLine("No logs found in artifacts/log");
             }
-            Console.WriteLine($"Copying TestResults/**/*.dmp to {HELIX_WORKITEM_UPLOAD_ROOT}/");
+            Console.WriteLine($"Copying TestResults/**/Sequence*.xml to {HELIX_WORKITEM_UPLOAD_ROOT}/");
```

Comment:
So, what are Sequence*.xml files❔

Reply:
You can look at them to see what tests ran and which didn't finish, its another way to look for hanging tests in the cases where there its not obvious from the logs.  There are some cases where sequence.xml doesn't get written out either, and in those cases you have to go all the way down to the detailed diagnostic logs: vstest.log

Reply:
We shouldn't have to go digging into either of these files normally, but it gives us the most diagnostics we can get right now for crashes/hangs